tests/langchain/test_langchain_model_export.py

```diff
@@ -1503,49 +1504,98 @@ def _call(self, messages, stop, run_manager, **kwargs):  # pylint: disable=signa
     )
 
     pyfunc_loaded_model = mlflow.pyfunc.load_model(model_info.model_uri)
-    assert pyfunc_loaded_model.predict(input_example) == [
-        (
-            "system: You are a helpful assistant.\n"
-            "ai: What would you like to ask?\n"
-            "human: Who owns MLflow?"
-        )
-    ]
+    with mock.patch("time.time", return_value=1677858242):
+        assert pyfunc_loaded_model.predict(input_example) == [
+            {
```

Comment:
Can we make this dictionary a variable as it's reused several times, the test looks quite long

Reply:
Done - great catch, sorry for the clutter!